geode-web/src/test/java/org/apache/geode/management/internal/cli/commands/ConnectCommandWithSSLTest.java

```diff
@@ -265,7 +271,26 @@ public void connectWithHttpSSL() throws Exception {
         "security-properties-file", sslConfigFile.getAbsolutePath());
     assertThat(gfsh.isConnected()).isFalse();
 
-    // can conect to http
+    // can connect to http
```

Comment:
The comments on the cases above don't match the assertions, e.g. "// can connect to jmx" is not consistent with "assertThat(gfsh.isConnected()).isFalse()"

Reply:
Fixed